packages/create-vite/solid-js/src/App.jsx

```diff
@@ -0,0 +1,34 @@
+import { useSignal } from 'solid-js'
+import solidLogo from './assets/solid.svg'
+import './App/module.css'
+
+function App() {
+  const [count, setCount] = useSignal(0)
+
+  return (
+    <div className="App">
+      <div>
+        <a href="https://vitejs.dev" target="_blank">
+          <img src="/vite.svg" className="logo" alt="Vite logo" />
+        </a>
+        <a href="https://solidjs.com" target="_blank">
+          <img src={solidLogo} className="logo solid" alt="React logo" />
+        </a>
+      </div>
+      <h1>Vite + Solid</h1>
+      <div className="card">
+        <button onClick={() => setCount((count) => count + 1)}>
+          count is {count}
```

Comment:
```suggestion
          count is {count()}
```

Solid needs to call as a function